packages/odyssey-react-mui/src/Typography.tsx

```diff
@@ -93,6 +93,7 @@ export type TypographyProps = {
   | "ariaLabelledBy"
   | "testId"
   | "translate"
+  | "role"
```

Comment:
This `role` is fine as `Typography` is an internal component.

Reply:
addressed with `type PublicTypographProps = Omit<TypographyProps, "role">;`